test/plugin/test_out_sentry.rb

```diff
@@ -42,28 +43,45 @@ def test_emit
     d1 = create_driver(CONFIG, 'input.app1_error')
     emit_level = 'warning'
     emit_message = 'error has occoured.'
+    emit_timestamp = '2015-10-30T08:25:48Z'
+    emit_time_spent = '13'
+    emit_logger = 'zaphod'
+    emit_server_name = 'bc40a4be7b2d'
+    emit_culprit = 'whodonit'
+    emit_release = 'wingnut'
     emit_extra = {'foo' => {'array' => [1,2,3]}, 'hash' => {'nest' => 'data'}}
     d1.run do
       d1.emit({
         'level' => emit_level,
         'message' => emit_message,
-        'something' => emit_extra
+        'something' => emit_extra,
+        'tags' => {'cool' => 'bar'},
+        'timestamp' => emit_timestamp,
+        'logger' => emit_logger,
+        'server_name' => emit_server_name,
+        'release' => emit_release,
+        'time_spent' => emit_time_spent,
+        'culprit' => emit_culprit
```

Comment:
so to clarify, i'll 
* rename `stub_endpoint` to `stub_post`
* create a new `stub_response`
* revert `test_emit` back to the old tests but call `stub_post` now
* add a new `test_emit_mock` method
* make `test_emit_mock` similar to the current test